kythe/go/extractors/config/wrapper/gradle_build_modifier_test.go

```diff
@@ -16,6 +16,10 @@ const testDataDir = "testdata"
 
 var multipleNewLines = regexp.MustCompile("\n{2,}")
 
+// filesEqual compares two strings by collapsing irrelevant whitespace.
```

Comment:
It collapses newlines, what about spaces/tabs? 
(I don't know if newlines are allowed inside string literals, which I guess could be an issue for 0x20)

Reply:
So it only collapses whitespace at the beginning/end of lines, which should be unnecessary.  I guess if there's a multiline string it could be a problem if the language being compared supports that (gradle and xml I think do not, so so far it hasn't come up)

Reply:
Well, XML supports `<![CDATA[…]>` sections, which can span multiple lines, but I would be slightly surprised to find a lot of those in build configs.